test/user.test.js

```diff
@@ -1763,6 +1763,367 @@ describe('User', function() {
     });
   });
 
+  describe('Email Update', function() {
+    describe('User changing email property', function() {
+      var usersId;
+      var currentEmailCredentials = { email: 'm@c.com', password: 'bar' };
+      var updatedEmailCredentials = { email: 'n@d.com', password: 'bar' };
+
+      beforeEach(function(done) {
```

Comment:
The hook should be named, see https://github.com/strongloop/loopback-contributor-docs/blob/master/style-guide.md#hooks. Consider also https://github.com/strongloop/loopback-contributor-docs/blob/master/style-guide.md#layout-of-test-files